src/Deployment/source.extension.vsixmanifest

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="eb2680f2-4e63-44a8-adf6-2e667d9f689c" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
-    <DisplayName>Roslyn Insiders</DisplayName>
+    <Identity Id="f6d4ae9d-5ca3-4e0b-9035-9457cccf53fa" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
```

Comment:
Note - I changed this because it was previously the same as the one in `src\Setup\Deployment\Current\source.extension.vsixmanifest`.

Do we even need this vsix anymore, or should it just be deleted in favor of the ones under src\Setup\Deployment?

@jmarolf?